mesa/examples/basic/boltzmann_wealth_model/app.py

```diff
@@ -1,3 +1,10 @@
+import os.path
+import sys
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../../"))
```

Comment:
We need some policy if we want to include this in every example, or handle it differently.

Reply:
it simply should not be there and we need to find a cleaner way to run these examples from the command line. Not critical for 3.0 but a prioirty afterwards